gprofiler/profilers/python.py

```diff
@@ -340,6 +342,16 @@ def _should_skip_process(self, process: Process) -> bool:
             action="store_true",
             help="Enable PyPerf in verbose mode (max verbosity)",
         ),
+        ProfilerArgument(
+            name="--python-pyspy-processes",
+            dest="python_pyspy_processes",
+            nargs="+",
+            type=int,
+            help="List of processes (by PID) to profile with py-spy."
+            "This option forces gProfiler to profile given processes with py-spy, even if"
+            "they are not recognized by gProfiler as Python processes."
+            "Note - gProfiler assumes that the given processes are alive and running forever.",
```

Comment:
I don't like the use of "forever" here
@Jongy what other word could fit?

Reply:
```suggestion
            "Note - gProfiler assumes that the given processes are kept running as long as gProfiler runs.",
```